screens/Home/index.tsx

```diff
@@ -70,12 +70,12 @@ export default function HomeScreen() {
     }, [isAuthenticated, refetchPoints]),
   )
   const { isPlaceholderData } = useMySignal()
-
   const handleWebsiteOpen = async () => {
-    await WebBrowser.openBrowserAsync(
-      "https://7axab-zyaaa-aaaao-qjv7a-cai.icp0.io/",
-    )
+    if (process.env.POINTS_CANISTER_URL) {
+      await WebBrowser.openBrowserAsync(process.env.POINTS_CANISTER_URL || "")
```

Comment:
since you checked that the env variable is valid, there's no need to add an empty string as a fallback. Consider creating a variable for `process.env.POINTS_CANISTER_URL` since it is being used two times